platform/mds/mds/src/test/java/org/motechproject/mds/it/service/EntityServiceContextIT.java

```diff
@@ -215,6 +215,36 @@ public void shouldSaveEntityWithGivenLookups() throws IOException {
     }
 
     @Test
+    public void shouldAddNewFieldForLookupAndSaveEntity() throws IOException {
+        EntityDto entityDto = new EntityDto();
+        entityDto.setName("myEntity");
+
+        Map<String, Object> values = new HashMap<>();
+        values.put("path", "indexes.0.$addField");
+        values.put("advanced", true);
+        DraftData draftData = new DraftData();
+        draftData.setEdit(true);
+        draftData.setValues(values);
+        entityDto = entityService.createEntity(entityDto);
+
+        FieldDto testField = FieldTestHelper.fieldDto("firstField", Boolean.class);
+        entityService.addFields(entityDto, Collections.singletonList(testField));
+        List<LookupFieldDto> lookupFieldDtos = lookupFieldsFromNames("firstField");
```

Comment:
why adding here lookupField ? Just testing adding this without any field in lookup